### PR TITLE
HUE-9619 [Deprecation Warning] Importing from django.core.urlresolver…

### PR DESCRIPTION
**Pull request by agl29**
…s is deprecated

### CODE REVIEW

File: apps/proxy/src/proxy/views.py [hunk 1]
```diff
@@ -31,8 +31,8 @@
 from urllib.request import Request, urlopen
 from urllib.parse import urlencode, urlparse, urlunparse
 
-from django.core import urlresolvers
 from django.http import HttpResponse
+from django.urls import resolvers
```
Comment (romainr) [inline apps/proxy/src/proxy/views.py:35]: Please keep the new line separator (local libs != desktop libs)

File: desktop/core/src/desktop/auth/backend.py [hunk 2]
```diff
@@ -46,10 +46,10 @@
 
 import django.contrib.auth.backends
 from django.contrib import auth
-from django.core.urlresolvers import reverse
 from django.core.exceptions import ImproperlyConfigured, PermissionDenied
 from django.http import HttpResponseRedirect
 from django.forms import ValidationError
+from django.urls import reverse
```
Comment (romainr) [inline desktop/core/src/desktop/auth/backend.py:52]: Same everywhere

**Review by romainr — APPROVED** [review]

**Review by romainr — APPROVED** [review]

File: apps/jobsub/src/jobsub/models.py [hunk 3]
```diff
@@ -18,8 +18,8 @@
 from builtins import str
 import logging
 
-from django.core import urlresolvers
 from django.db import models
+from django.urls import resolvers, reverse
```
Comment (romainr) [inline apps/jobsub/src/jobsub/models.py:22]: Can we remove `resolvers` if not used?

File: desktop/core/src/desktop/middleware.py [hunk 4]
```diff
@@ -38,9 +38,9 @@
 from django.contrib import messages
 from django.contrib.auth import REDIRECT_FIELD_NAME, BACKEND_SESSION_KEY, authenticate, load_backend, login
 from django.contrib.auth.middleware import RemoteUserMiddleware
-from django.core import exceptions, urlresolvers
+from django.core import exceptions
 from django.http import HttpResponseNotAllowed, HttpResponseForbidden
-from django.urls import resolve
+from django.urls import resolve, resolvers
```
Comment (romainr) [inline desktop/core/src/desktop/middleware.py:43]: Same everywhere, looks like `resolvers` is not used

**Review by romainr — APPROVED** [review]

File: apps/jobsub/src/jobsub/models.py (hunk 3 above)
Comment (agl29) [reply apps/jobsub/src/jobsub/models.py]: Done

File: desktop/core/src/desktop/middleware.py (hunk 4 above)
Comment (agl29) [reply desktop/core/src/desktop/middleware.py]: Done

**Pull request merged**